crates/core/src/document/html/engine.rs

```diff
@@ -870,13 +872,16 @@ impl Engine {
                                     };
                                     plan.space_out(style.letter_spacing);
 
+                                    let ruby_chars = style.ruby.clone().map(|r| r.chars().count());
+                                    let ruby_chars_use = ruby_chars.map(|c| std::cmp::min(c.div_ceil(text_len), (c - ruby_start_index).div_ceil(text_len)));
```

Comment:
Isn't `std::cmp::min(c.div_ceil(text_len), (c - ruby_start_index).div_ceil(text_len))` equal to `(c - ruby_start_index).div_ceil(text_len)`?

Reply:
True, that can be simplified here.

Reply:
https://github.com/antontarasenko/awesome-economics/commit/6b9f22e8841a62735168b388b4787b5d9d541793

Reply:
![bulk-sponsorships-template.csv](https://github.com/user-attachments/files/16555472/bulk-sponsorships-template.csv)

Reply:
> Isn't `std::cmp::min(c.div_ceil(text_len), (c - ruby_start_index).div_ceil(text_len))` equal to `(c - ruby_start_index).div_ceil(text_len)`?